templates/default/capacity-scheduler.xml.erb

```diff
@@ -33,7 +33,7 @@
 
   <property>
     <name>yarn.scheduler.capacity.resource-calculator</name>
-    <value>org.apache.hadoop.yarn.util.resource.DefaultResourceCalculator</value>
+    <value>org.apache.hadoop.yarn.util.resource.DominantResourceCalculator</value>
```

Comment:
Great.

On 2015-11-10 15:43, marcbux wrote:
>
> In templates/default/capacity-scheduler.xml.erb 
> <https://github.com/hopshadoop/apache-hadoop-chef/pull/6#discussion_r44413332>:
>
> > @@ -33,7 +33,7 @@
> >
> >    <property>
> >      <name>yarn.scheduler.capacity.resource-calculator</name>
> > -    <value>org.apache.hadoop.yarn.util.resource.DefaultResourceCalculator</value>
> > +    <value>org.apache.hadoop.yarn.util.resource.DominantResourceCalculator</value>
> Good point. I'll do as you suggested, but I'll also add a check that 
> will set hiway's vcores parameters to 1 if the 
> DominantResourceCalculator is not used. I'll create a new pull request 
> as soon as I'm done with it.
> …
> On Tue, Nov 10, 2015 at 12:51 PM, Jim Dowling 
> <notifications@github.com> wrote: In 
> templates/default/capacity-scheduler.xml.erb <#6 (comment) 
> <https://github.com/hopshadoop/apache-hadoop-chef/pull/6#discussion_r44397559>> 
> : > @@ -33,7 +33,7 @@ > > <property> > 
> <name>yarn.scheduler.capacity.resource-calculator</name> > - 
> <value>org.apache.hadoop.yarn.util.resource.DefaultResourceCalculator</value> 
> > + 
> <value>org.apache.hadoop.yarn.util.resource.DominantResourceCalculator</value> 
> The DominantResourceCalculator is not available in hadoop 2.4, 
> http://hadoop.apache.org/docs/r2.4.1/hadoop-yarn/hadoop-yarn-site/CapacityScheduler.html, 
> so you can override this in your yml files. Can you make this a 
> parameter, and keep the defaultResourceCalculator? — Reply to this 
> email directly or view it on GitHub 
> <https://github.com/hopshadoop/apache-hadoop-chef/pull/6/files#r44397559>. 
>
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/hopshadoop/apache-hadoop-chef/pull/6/files#r44413332>.
>

-- 
-----
Dr. Jim Dowling
Senior Researcher, SICS - Swedish ICT
Associate Prof, KTH - Royal Institute of Technology